examples/single_column_os_papa_simulation.jl

```diff
@@ -62,7 +62,7 @@ set!(ocean.model, T=ECCOMetadatum(:temperature),
 atmosphere = JRA55PrescribedAtmosphere(longitude = λ★,
                                        latitude = φ★,
                                        end_date = DateTime(1990, 1, 31), # Last day of the simulation
-                                       backend = JRA55NetCDFBackend(30))
+                                       backend  = InMemory())
```

Comment:
@glwagner, I think the DO NOT MERGE label was for this change here. Because this is a hack for #399.
Otherwise I agree, better merge the examples that work and then fix more later.